pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -111,7 +112,26 @@ const (
 )
 
 var (
-	supportedLanguages = []language{java, js, python, dotnet, ruby}
+	supportedLanguages = []language{
+		java,
+		js,
+		python,
+		dotnet,
+		ruby,
+	}
+
+	// languageVersions defines the major library versions we consider "default" for each
+	// supported language. If not set, we will default to "latest", see defaultLibVersion.
+	//
+	// If this language does not appear in supportedLanguages, it will not be injected.
+	languageVersions = map[language]string{
+		java:   "v1", // https://datadoghq.atlassian.net/browse/APMON-1064
```

Comment:
| lang | version | docker hub link | ok | 
| - | - | - | - |
| java | v1 | https://hub.docker.com/layers/datadog/dd-lib-java-init/v1/images/sha256-580b6bba541e5793559b8eb177225dd42ee77e01b5b06260ba2dabdf7a5781aa | ✅ |
| dotnet | v2 | https://hub.docker.com/layers/datadog/dd-lib-dotnet-init/v2/images/sha256-409f915272bbf444e10ae4ee42c8deb6a7f068f2929010fc0ef07a6623f21ccf | ✅ |
| python | v2 | https://hub.docker.com/layers/datadog/dd-lib-python-init/v2/images/sha256-37c2f1d805f17ad66b0761735095bb85e3f7679130eb68242e870dc8795d6689 | ✅ |
| ruby | v2 | https://hub.docker.com/layers/datadog/dd-lib-ruby-init/v2/images/sha256-12337162d0fc65f82f10a97430ffd6b63da8e830385c01e2750a639a35913db3 | ✅ |
| js | v5 | https://hub.docker.com/layers/datadog/dd-lib-js-init/v5/images/sha256-8e045be081d24264e20d63d16de960c5e3ba61fddfc59d2fdaef9ba241ce9ea3 | ✅ |
| php | v2 | - | 🟥  |

Reply:
The php thing is probably fine since it's not an "available language" anyway.